spec/integration_spec.rb

```diff
@@ -16,33 +16,33 @@ class Comment < ActiveRecord::Base; end
   end
 
   describe 'logging' do
-    context 'when the migration logging is enabled' do
+    context 'when the migration logging is disabled' do
       around(:each) do |example|
         original_verbose = ActiveRecord::Migration.verbose
-        ActiveRecord::Migration.verbose = true
+        ActiveRecord::Migration.verbose = false
         example.run
         ActiveRecord::Migration.verbose = original_verbose
       end
 
       it 'sends the output to the stdout' do
         expect do
           ActiveRecord::Migrator.new(direction, migration_fixtures, 1).migrate
-        end.to output.to_stdout
+        end.to_not output.to_stdout
       end
     end
 
-    context 'when the migration logging is disabled' do
+    context 'when the migration logging is enabled' do
       around(:each) do |example|
         original_verbose = ActiveRecord::Migration.verbose
-        ActiveRecord::Migration.verbose = false
+        ActiveRecord::Migration.verbose = true
         example.run
         ActiveRecord::Migration.verbose = original_verbose
       end
 
       it 'sends the output to the stdout' do
         expect do
           ActiveRecord::Migrator.new(direction, migration_fixtures, 1).migrate
-        end.to_not output.to_stdout
+        end.to output.to_stdout
```

Comment:
Yeah.  This just slipped past me. I was chasing transient failures, and there is one related to these two specs. It seems to be something within AR and not at all related to Departure, but, but if the spec that outputs to stdout runs before the spec that should not output to stdout, that second spec can fail. Switching the order, and running the specs in defined order instead of random order proved it. I never figured out a solution, and forgot to switch them back.